vumi/blinkenlights/metrics.py

```diff
@@ -75,7 +75,8 @@ def stop_polling(self):
         Stop the metric polling and publishing task.
         """
         if self._task:
-            self._task.stop()
+            if self._task.running:
+                self._task.stop()
```

Comment:
Seems a bit unrelated to this issue, although seems like a good idea? Just checking it didn't sneak in unintentionally.